pyspeckit/spectrum/classes.py

```diff
@@ -181,14 +180,12 @@ def __init__(self, filename=None, filetype=None, xarr=None, data=None,
                 self.data = np.ma.masked_where(np.isnan(self.data) + np.isinf(self.data), self.data)
                 self.error = np.ma.masked_where(np.isnan(self.data) + np.isinf(self.data), self.error)
 
-        # it is very important that this be done BEFORE the spectofit is set!
-        self._sort()
-
```

Comment:
please note this comment - this is one of the cases where the merge went the wrong way.  There will be hard-to-understand crashes if `_sort` is done after `specfit` is initialized.